test/test_adaptive_thresholding.py

```diff
@@ -0,0 +1,55 @@
+"""this module tests the adaptive thresholding class"""
+import sys
+import os
+
+ROOT_DIR = os.path.abspath(os.curdir)
+sys.path.append(ROOT_DIR)
```

Comment:
Why is this necessary?

Reply:
py.test will error saying it can’t find the vein_detection class (atleast on Mac OS) when this is missing

Reply:
Try setting the PYTONPATH environment variable (not sure how that's done on Mac OS) to your R2D2 python build folder and importing using `from modules.MODULE_NAME.module.FILE_NAME import MY_CLASS` . Did you follow the python build wiki?

Reply:
We haven’t implemented the python build yet, will look into it. Thanks!

Reply:
@nicovanbentum I looked into this with Isha sadly it won't work using those described steps. I attached a picture showing the error.

<img width="1207" alt="Schermafbeelding 2019-07-01 om 11 23 30" src="https://user-images.githubusercontent.com/31623036/60426577-3598a600-9bf5-11e9-9c3b-25588691719d.png">

Reply:
Path wasn't set correctly, resolved now.